modal/__init__.py

```diff
@@ -74,6 +74,7 @@
     "forward",
     "is_local",
     "method",
+    "stop_fetching_inputs",
```

Comment:
If this is exposed in the top-level namespace, won't most users miss that it is `experimental`? Any cost to asking beta testers import it from the full path?

Reply:
It's not even imported anywhere, so this would not have worked